cs3/ocm/core/v1beta1/ocm_core_api.proto

```diff
@@ -109,3 +120,55 @@ message CreateOCMCoreShareResponse {
   // REQUIRED.
   cs3.types.v1beta1.Timestamp created = 4;
 }
+
+message UpdateOCMCoreShareRequest {
+  // OPTIONAL.
+  // Opaque information.
+  cs3.types.v1beta1.Opaque opaque = 1;
+  // REQUIRED.
+  // Unique ID to identify the share at the consumer side.
+  string ocm_share_id = 2;
+  // OPTIONAL.
+  // Description for the share.
+  string description = 3;
+  // OPTIONAL.
+  // Consumer specific identifier of the user or group the provider wants to share the resource with.
+  // TODO: this field needs to represent either a user or group in the future, not only a user.
+  cs3.identity.user.v1beta1.UserId share_with = 4;
```

Comment:
do we expect this to change? I expect that you cannot change the user of a share, like normal shares. instead you unshare and share with a new user. same will apply for groups in future

Reply:
Absolutely correct - when going through all fields from `Create` I overlooked that this is also to be dropped